website/source/docs/secrets/pki/index.html.md

```diff
@@ -260,6 +339,439 @@ If you get stuck at any time, simply run `vault path-help pki` or with a subpath
+    it is `internal` the private key will not be returned and *cannot be
+    retrieved later*. Distribution points use the values set via `config/urls`.
+  </dd>
+
+  <dt>Method</dt>
+  <dd>POST</dd>
+
+  <dt>URL</dt>
+  <dd>`/pki/config/ca/generate/root/[exported|internal]`</dd>
+
+  <dt>Parameters</dt>
+  <dd>
+    <ul>
+      <li>
+        <span class="param">common_name</span>
+        <span class="param-flags">required</span>
+        The requested CN for the certificate.
+      </li>
+      <li>
+        <span class="param">alt_names</span>
+        <span class="param-flags">optional</span>
+        Requested Subject Alternative Names, in a comma-delimited list. These
+        can be host names or email addresses; they will be parsed into their
+        respective fields.
+      </li>
+      <li>
+        <span class="param">ip_sans</span>
+        <span class="param-flags">optional</span>
+        Requested IP Subject Alternative Names, in a comma-delimited list.
+      </li>
+      <li>
+      <span class="param">ttl</span>
+      <span class="param-flags">optional</span>
+        Requested Time To Live (after which the certificate will be expired).
+        This cannot be larger than the mount max (or, if not set, the system
+        max).
+      </li>
+      <li>
+      <span class="param">format</span>
+      <span class="param-flags">optional</span>
+        Format for returned data. Can be `pem` or `der`; defaults to `pem`. If
+        `der`, the output is base64 encoded.
+      </li>
+      <li>
+        <span class="param">key_type</span>
+        <span class="param-flags">optional</span>
+        Desired key type; must be `rsa` or `ec`. Defaults to `rsa`.
+      </li>
+      <li>
+        <span class="param">key_bits</span>
+        <span class="param-flags">optional</span>
+        The number of bits to use. Defaults to `2048`. Must be changed to a
+        valid value if the `key_type` is `ec`.
+      </li>
+      <li>
+        <span class="param">max_path_length</span>
+        <span class="param-flags">optional</span>
+        If set, the maximum path length to encode in the generated certificate.
+        Defaults to `-1`, which means no limit. A limit of `0` means a literal
+        path length of zero.
+      </li>
+    </ul>
+  </dd>
+
+  <dt>Returns</dt>
+  <dd>
+
+    ```javascript
+    {
+      "lease_id": "",
+      "renewable": false,
+      "lease_duration": 21600,
+      "data": {
+        "certificate": "-----BEGIN CERTIFICATE-----\nMIIDzDCCAragAwIBAgIUOd0ukLcjH43TfTHFG9qE0FtlMVgwCwYJKoZIhvcNAQEL\n...\numkqeYeO30g1uYvDuWLXVA==\n-----END CERTIFICATE-----\n",
+        "issuing_ca": "-----BEGIN CERTIFICATE-----\nMIIDzDCCAragAwIBAgIUOd0ukLcjH43TfTHFG9qE0FtlMVgwCwYJKoZIhvcNAQEL\n...\numkqeYeO30g1uYvDuWLXVA==\n-----END CERTIFICATE-----\n",
+        "serial": "39:dd:2e:90:b7:23:1f:8d:d3:7d:31:c5:1b:da:84:d0:5b:65:31:58"
+        },
+      "auth": null
+    }
+    ```
+
+  </dd>
+</dl>
+
+### /pki/config/ca/sign
```

Comment:
It seems odd that this is under `/pki/config` maybe these should just be moved to `/pki/ca/`?